mlrun/projects/pipelines.py

```diff
@@ -732,18 +732,8 @@ def get_state(run_id, project=None):
         return ""
 
     @staticmethod
-    def wait_for_completion(run_id, project=None, timeout=None, expected_statuses=None):
-        pass
-
-    @staticmethod
-    def get_run_status(
-        project,
-        run,
-        timeout=None,
-        expected_statuses=None,
-        notifiers: mlrun.utils.notifications.CustomNotificationPusher = None,
-        **kwargs,
-    ):
+    def wait_for_completion(run, project=None, timeout=None, expected_statuses=None):
+        # local runner does not have a wait_for_completion method because we already wait in the `run` function
```

Comment:
```suggestion
        # TODO: local runner blocks for the duration of the pipeline therefore usually there will be nothing to wait for. However, users may run functions with watch=False and then it can be useful to wait for the runs here.
```